staging/src/k8s.io/apiserver/pkg/authorization/cel/mapper.go

```diff
@@ -19,10 +19,9 @@ package cel
 import (
 	"context"
 	"fmt"
-	"reflect"
 
+	celgo "github.com/google/cel-go/cel"
 	authorizationv1 "k8s.io/api/authorization/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
```

Comment:
nit: just noticed this isn't mapping, it's matching. I'm guessing this was mimicking the authn cel code which _was_ mapping. If it's easy, can rename to matcher.go, `CELMatcher`, the WebhookAuthorizer instance variable to `celMapper`, tweak comments like `eval evaluates the given SubjectAccessReview cel matchCondition expression`, etc. If it's not easy I'll take a follow-up for 1.30.